src/Azure.Functions.Cli/Helpers/PythonHelpers.cs

```diff
@@ -125,21 +125,21 @@ public static async Task<WorkerLanguageVersionInfo> GetEnvironmentPythonVersion(
                 return await GetVersion(pythonDefaultExecutablePath);
             }
 
-            var pythonGetVersionTask = GetVersion("python");
+            var pyGetVersionTask = GetVersion("py"); // Windows Python Launcher
```

Comment:
Maybe it's fine, but this will almost always be a useless operation in a non-Windows environment. I wonder if we should do
```csharp
var pyGetVersionTask = IsWindows ? GetVersion("py") : Task.FromResult<WorkerLanguageVersionInfo>(null);
```

or something like that.